controllers/new.go

```diff
@@ -161,6 +168,11 @@ func (k *kubeResources) newGraphBuilderService(instance *cv1.UpdateService) *cor
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      name,
 			Namespace: instance.Namespace,
+			Annotations: map[string]string{
+				DescriptionAnnotation: "It exposes a directed acyclic graph that represent the valid updates. " +
+					"Each node in the graph is a release payload and each edge is a valid transition. " +
+					"See https://github.com/openshift/cincinnati/blob/master/docs/design/cincinnati.md#graph-builder for more details",
```

Comment:
Hmm, we don't front the graph-builder Service with a Route or anything, right?  I think it exists only to support ServiceMonitor scraping of graph-builder Prometheus metrics?

Reply:
The service proxies 2 ports: `8080` and `9080`.
`8080` is for [the main service port](https://github.com/openshift/cincinnati/blob/5092feec15ecf8e558cd43f7d74a204981751713/graph-builder/src/config/settings.rs#L41-L43) and IIRC, it provides the graphs [here](https://github.com/openshift/cincinnati/blob/5092feec15ecf8e558cd43f7d74a204981751713/graph-builder/src/main.rs#L126).

I think `9080` [the metrics port ](https://github.com/openshift/cincinnati/blob/5092feec15ecf8e558cd43f7d74a204981751713/graph-builder/src/main.rs#L107) that you meant.

It is true that there is no route in front of the service. The word `exposing` might be confusing but it is also used here:

```console
oc explain svc.spec.ports
KIND:       Service
VERSION:    v1

FIELD: ports <[]ServicePort>

DESCRIPTION:
    The list of ports that are exposed by this service. ...
```

Reply:
Maybe we should say something like:

> This Service exposes a client-agnostic update graph to other clients within the cluster.   This allows convenient in-cluster access to those graphs, and also allows platform monitoring to scrape graph-builder containers for Prometheus metrics.  See https://github.com/openshift/cincinnati/blob/master/docs/design/cincinnati.md#graph-builder for more details.

or something that is more specific about who we're exposing this to?

Reply:
Updated.